admin/formatter/class-metabox-formatter.php

```diff
@@ -81,6 +85,7 @@ private function get_defaults() {
 			'wordFormRecognitionActive' => YoastSEO()->helpers->language->is_word_form_recognition_active( WPSEO_Language_Utils::get_language( get_locale() ) ),
 			'siteIconUrl'               => get_site_icon_url(),
 			'countryCode'	   			=> WPSEO_Options::get( 'semrush_country_code', false ),
+			'SEMrushLoginStatus'		=> $this->get_semrush_login_status( $options ),
```

Comment:
This alignment should be made with spaces. I see tabs are used instead. Same for line 87 (pre-existing issue).